packages/experimental/Button/src/CompoundButton/CompoundButton.styling.ts

```diff
@@ -0,0 +1,82 @@
+import { compoundButtonName, CompoundButtonTokens, CompoundButtonSlotProps, CompoundButtonProps } from './CompoundButton.types';
+import { Theme, UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { borderStyles, fontStyles } from '@fluentui-react-native/tokens';
+import { buttonStates, defaultButtonTokens } from '../ButtonTokens';
+import { ButtonTokens } from '../Button.types';
+
+export const stylingSettings: UseStylingOptions<CompoundButtonProps, CompoundButtonSlotProps, CompoundButtonTokens> = {
+  tokens: [
+    (t: Theme) => ({
+      ghost: {
+        secondaryContentTokens: {
+          variant: 'secondaryStandard',
+          color: t.colors.ghostSecondaryContent,
+          hovered: {
+            color: t.colors.ghostHoveredSecondaryContent,
+          },
+          focused: {
+            color: t.colors.ghostFocusedSecondaryContent,
+          },
+          pressed: {
+            color: t.colors.ghostPressedSecondaryContent,
+          },
+        },
+      } as ButtonTokens,
+    }),
+    defaultButtonTokens,
+    compoundButtonName,
+  ],
+  states: buttonStates,
+  slotProps: {
+    root: buildProps(
```

Comment:
I wonder if some of these could be shared with button. It seems like much of this is somewhat duplicated. Maybe something like the way borderStyles or fontStyles are defined.